SlicerDevelopmentToolboxUtils/mixins.py

```diff
@@ -717,7 +717,7 @@ def getDICOMValue(inputArg, tagName, default=""):
       elif type(inputArg) is str and os.path.isfile(inputArg):
         value = slicer.dicomDatabase.fileValue(inputArg, tagName)
       elif type(inputArg) is slicer.vtkMRMLScalarVolumeNode:
-        f = inputArg.GetStorageNode().GetFileName()
+        f = slicer.dicomDatabase.fileForInstance(inputArg.GetAttribute("DICOM.instanceUIDs").split(" ")[0])
```

Comment:
Shouldn't this `elif` be collapsed with the subsequent one?

Reply:
Also an option, but I actually prefer having the handling of each input data type listed separately.